packages/esm-patient-tests-app/src/test-results/trendline/range-selector.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { fireEvent, render, screen } from '@testing-library/react';
+import RangeSelector from './range-selector.component';
+
+describe('RangeSelector', () => {
+  const mockSetLowerRange = jest.fn();
+
+  beforeEach(() => {
+    mockSetLowerRange.mockClear();
+  });
+
+  it('should render all range tabs', () => {
+    const upperRange = new Date('2024-12-31');
+    render(<RangeSelector setLowerRange={mockSetLowerRange} upperRange={upperRange} />);
+
+    expect(screen.getByText('1 day')).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByRole('tab', { name: '1 day' })).toBeInTheDocument();
```

Prefer using accessible queries such as `getByRole` wherever possible https://testing-library.com/docs/queries/about#priority.